packages/angular/cli/models/architect-command.ts

```diff
@@ -350,7 +352,8 @@ export abstract class ArchitectCommand<
         return allProjectsForTargetName;
       }
 
-      throw new Error(`Could not determine a single project for the '${targetName}' target.`);
+      throw new Error(this.missingTargetError ||
```

Comment:
In this case there are multiple projects with the particular target and the target is a single instance target.  The missing error shouldn't apply here as it actually found several.